docs/migration_guide.md

```diff
@@ -14,61 +14,83 @@ Since the migration might alter the database schema and the settings of `harbor.
 - With the introduction of storage and artifact quotas in version 1.9.0, replication between version 1.9.0 and a previous version of Harbor does not work. You must upgrade all Harbor nodes to 1.9.0 if you have configured replication between them.
 
+
+
+
+
 ## Upgrading Harbor and Migrating Data
 
-1. Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
+#### Shutdown Harbor Service
 
-    ```sh
-    cd harbor
-    docker-compose down
-    ```
+Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
 
-2. Back up Harbor's current files so that you can roll back to the current version if necessary.
+```sh
+cd harbor
+docker-compose down
+```
 
-    ```sh
-    mv harbor /my_backup_dir/harbor
-    ```
+#### Backup Harbor Data
 
-    Back up database (by default in directory `/data/database`)
+Back up Harbor's current files so that you can roll back to the current version if necessary.
 
-    ```sh
-    cp -r /data/database /my_backup_dir/
-    ```
+```sh
+mv harbor /my_backup_dir/harbor
+```
+
+Back up database (by default in directory `/data/database`)
 
-3. Get the latest Harbor release package from Github:
-   [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases)
+```sh
+cp -r /data/database /my_backup_dir/
+```
 
-4. Before upgrading Harbor, perform migration first.  The migration tool is delivered as a docker image.
+#### Get Package of Latest Harbor Installer
+
+You can download it in this url [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases).
+
+#### Migrating Config file
+
+Before upgrading Harbor, perform migration first. The migration tool is delivered as a docker image.
+
+1. **Online installer** need pull the migration image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
 
-    You can pull the image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
     ```sh
     docker pull goharbor/harbor-migrator:[tag]
     ```
 
-    Alternatively, if you are using an offline installer package you can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+    **Offline installer** can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+
     ```sh
     tar zxf <offline package>
     docker image load -i harbor/harbor.[version].tar.gz
     ```
 
-5. If you are current version is v1.7.x or earlier, i.e. migrate config file from `harbor.cfg` to `harbor.yml`.
+2. **If current Harbor version is v1.7.x or earlier**, You need migrate config file from `harbor.cfg` to `harbor.yml`. So need mount both `harbor.cfg` and `harbor.yml` to run the command.
```

Comment:
**If your current Harbor version is v1.7.x or earlier**, you must migrate the configuration file from `harbor.cfg` to `harbor.yml`. You must mount both `harbor.cfg` and `harbor.yml` to be able to run the command.